src/convertToObject.js

```diff
@@ -6,7 +6,18 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const separateParams = sourceString.split(';');
+  const cssStyles = {};
+
+  separateParams.forEach((el) => {
+    const secondsSplit = el.split(':');
+
+    if (secondsSplit[0] && secondsSplit[1]) {
+      cssStyles[secondsSplit[0].trim()] = secondsSplit[1].trim();
+    }
+  });
```

Comment:
[CODE STYLE] - All chained methods called on the same level should have the same indentation. This makes code much more readable. Although there are no chained methods here, the principle of clear and readable code applies. Consider using array iteration methods like `map` or `filter` to transform the string into an object in a more functional programming style.